gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -87,7 +91,12 @@ def main(args):
     apply_expected_variant_partition_hint = args.apply_expected_variant_partition_hint
     use_pops = args.use_pops
     use_weights = args.use_weights
+    use_v2_release_mutation_ht = args.use_v2_release_mutation_ht
     custom_vep_annotation = args.custom_vep_annotation
+    gerp_lower_cutoff = args.gerp_lower_cutoff
+    gerp_upper_cutoff = args.gerp_upper_cutoff
+
+    pops = POPS if use_pops else ()  # TODO: fix throughout code
```

Comment:
Is this still a TODO or has it all been fixed?

Reply:
all fixed